protocol-impl/src/main/java/io/zeebe/protocol/impl/encoding/MsgPackConverter.java

```diff
@@ -161,6 +161,19 @@ private static void convert(
     }
   }
 
+  public static Map<String, String> convertToStringMap(DirectBuffer buffer) {
+    final byte[] msgpackBytes = BufferUtil.bufferAsArray(buffer);
+    final byte[] jsonBytes = convertToJsonBytes(new ByteArrayInputStream(msgpackBytes));
+
+    final TypeReference<HashMap<String, String>> typeRef =
+        new TypeReference<HashMap<String, String>>() {};
+    try {
+      return JSON_OBJECT_MAPPER.readValue(jsonBytes, typeRef);
```

Comment:
Maybe we can create a new method out of that? Which takes the buffer and a typeref?